### ISSUE TITLE
Good Work! But how to do model selection without validation set?

### ISSUE THREAD

**Issue by Sissuire** [nie-lang/DeepRectangling#3]
A good work! 

I have checked your dataset, and I did not find the validation set. And I also noticed in your code you have adopted the full training set for training. So, how can I do model selection without the explicit validation set? Or, how you determine your pretrained model in your training phase?

**Comment by nie-lang** [comment]
We select the pre-trained model by directly evaluating different models on the testing set and testing the generalization capability on other datasets.

**Comment by Sissuire** [comment]
@nie-lang Thanks :) 
So, in the paper, accounting for the intra-dataset and cross-dataset comparison, is it the same model for all the tables and figures? or ~ they are different (i.e., one for quantitative comparison in table 1~3, and the other for the good generalization in figure 8/11/12).

**Comment by nie-lang** [comment]
Yes, we use the same model for all experiments.

**Comment by Sissuire** [comment]
@Sissuire Thanks for your kindly reply.
